app/vep/models/pipeline_model.py

```diff
@@ -87,9 +88,10 @@ def create_config_ini_file(self, dir):
             with open(os.path.join(dir, "config.ini"), "w") as ini_file:
                 ini_file.write(config_ini)
             return ini_file
+        except HTTPError:
```

Comment:
Why are we raising HTTP errors in file open action?

Reply:
Sorry, not needed here, removed it (was meant for forwarding httperrors from line 72).